src/Parser/ShadowPriest/Modules/Spells/Voidform.js

```diff
@@ -0,0 +1,255 @@
+      if(averageVoidformDuration < (this.voidformStatistics.reduce((p, c) => p += c.stacks.length, 0) / (Object.keys(this._voidformStatistics).length - 1)) - 5){
+
+        this._voidformStatistics[this._previousVoidformCast.timestamp] = {
+          ...this._voidformStatistics[this._previousVoidformCast.timestamp],
+          excluded: true,
+        };
+      }
+    }
+
+    // set end to last voidform of the fight:
+    if(this._voidformStatistics[this._previousVoidformCast.timestamp].ended === undefined){
+      this._voidformStatistics[this._previousVoidformCast.timestamp] = {
+        ...this._voidformStatistics[this._previousVoidformCast.timestamp],
+        ended: this.owner._timestamp,
+      };
+    }
+  }
+
+  suggestions(when) {
+    const { selectedCombatant, fightDuration } = this.owner;
+    const uptime = selectedCombatant.getBuffUptime(SPELLS.VOIDFORM_BUFF.id) / (fightDuration - this.dispersion.uptime);
+
+
+    when(uptime).isLessThan(0.85)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span>Your <SpellLink id={SPELLS.VOIDFORM.id} /> uptime can be improved. Try to maximize the uptime by using <SpellLink id={SPELLS.VOID_TORRENT.id} />, <SpellLink id={SPELLS.MINDBENDER.id} /> at optimal stacks.
+
+            <br /><br />
+            Managing your <SpellLink id={SPELLS.VOIDFORM.id} />s is a large part of playing shadow. The recommended way is to try to keep your <SpellLink id={SPELLS.VOIDFORM.id} /> cycles to around 60 seconds each, meaning you will have access to 1 <SpellLink id={SPELLS.VOID_TORRENT.id} /> & 1 <SpellLink id={SPELLS.MINDBENDER.id} /> each <SpellLink id={SPELLS.VOIDFORM.id} />.
+            <br /><br />
+            A good practice is to use <SpellLink id={SPELLS.VOID_TORRENT.id} /> shortly after entering <SpellLink id={SPELLS.VOIDFORM.id} />. <br/>
+            At around 28-33 <SpellLink id={SPELLS.VOIDFORM.id} /> stacks, use <SpellLink id={SPELLS.MINDBENDER.id} />.
+
+            <br /><br />
+            <SpellLink id={SPELLS.DISPERSION.id} /> can be used to synchronize your cooldowns back in order or in case of an emergency if you are about to fall out of <SpellLink id={SPELLS.VOIDFORM.id} /> and you have an <SpellLink id={SPELLS.MINDBENDER.id} /> active.
+          </span>)
+          .icon(SPELLS.VOIDFORM_BUFF.icon)
+          .actual(`${formatPercentage(actual)}% Voidform uptime`)
+          .recommended(`>${formatPercentage(recommended)}% is recommended`)
+          .regular(recommended).major(recommended - 0.10);
+      });
+
+    when(this.voidformAverageDuration).isLessThan(50)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span>Your <SpellLink id={SPELLS.VOIDFORM.id} /> uptime can be improved. Try to maximize the uptime by using <SpellLink id={SPELLS.VOID_TORRENT.id} />, <SpellLink id={SPELLS.MINDBENDER.id} /> at optimal stacks.
```

Comment:
This suggestion has the same description as the previous suggestion, shouldn't this be different?